cpp/ql/test/library-tests/ir/ssa/ssa.c

```diff
@@ -0,0 +1,8 @@
+struct Foo { int x[2]; };
+void named_designators() {
+  struct Foo foo = {.x[0] = 1234, .x[1] = 5678};
+}
+
+void repeated_designators() {
+  int x[1] = {[0] = 1234, [0] = 5678};
+}
```

Comment:
Do we already have tests for these kinds of initialization without repetition?

Reply:
I didn't find anything in any tests that did any IR printing, at least. I can add some more tests, though

Reply:
Yes please 😄

Reply:
3daefa8b9e26ef031dd903f5c48dff03fe185869 adds more tests.

Reply:
Thanks.